06-ssb-read.js

```diff
@@ -1,33 +1,32 @@
-var SSB = require('ssb-db')
 var Paramap = require('pull-paramap')
 var pull = require('pull-stream')
-var db = SSB(null, {}, null, '/tmp/bench-ssb-legacy_ssb/')
-db.ready.set(true)
 
-var a = []
+var createSsb = require('secret-stack')(require('ssb-config'))
+  .use(require('ssb-db'))
+var db = createSsb({ path: '/tmp/bench-ssb-legacy_ssb/' })
+
 var i = 0
 
 pull(
-//  db.createLogStream({reverse: true, keys: true, values: false, meta: true}),
-  db.time.read({keys: true, values: false, gt: 0}),
+  db.createLogStream({reverse: true, keys: true, values: false, seqs: true, meta: true}),
```

Comment:
this must test something else @christianbundy or it got massively slower (3x) which I don't think happened.

Reply:
Hmm, this bit *should* be outside of the actual test itself, right? I wonder what it's returning that would slow down the test.

I used `createLogStream()` here because that's what was commented out, and `db.time.read()` isn't in our API anymore.